extensions/wikia/Chat2/ChatHelper.php

```diff
@@ -298,6 +298,11 @@ public static function formatLogEntry( $type, $action, $title, $forUI, $params,
 		return wfMsg( 'chat-' . $action . '-log-entry', $link, $time, $endon );
 	}
 
+	public static function onOtherBlockLogLink( &$otherBlockLinks, $target ) {
+		$otherBlockLinks[] = Linker::specialLink('ChatBanList');
```

Comment:
Should be `...  Linker::specialLink( 'ChatBanList' );` because of spacing conventions, no?